app/controllers/cars_controller.rb

```diff
@@ -83,10 +84,8 @@ def car_params
       params.require(:car).permit(:brand, :model, :engine, :fuel_type, :year, :license_plate, :content)
     end
 
-
     # Deny url to be edited, and deny access to other users cars
-    def check_user
-      @car = current_user.cars.find(params[:id])
-    end
-
+    # def check_user
+    #   @car = current_user.cars.find(params[:id])
+    # end
```

Comment:
Don't leave commented out code behind, in big projects people after you will not understand if this code is needed

Reply:
Actually I was planning to show on the demo what authentication I use for the cars and user id before cancancan gem do it for me.